### PR TITLE
Description restructuring

### PR DESCRIPTION
**Pull request by hmagarotto**
- Using SchemaDefinition for Error payloads and Subscriptions.
- Removed Resources Tree and API Design chapters.
- Moved HTTP Methods and Error Handling chapters to root.
- Moved up SDK chapter.
- Added Sandbox Chapter.

_internal staging: http://zenvia-openapi-spec-description-restructuring.k8s.eqx.zenvia360.com/_

### CODE REVIEW

File: spec/components/schemas/error/base.ts [hunk 1]
```diff
@@ -10,19 +10,21 @@ const base: SchemaObject = {
...
-      description: 'Error code.',
+      description: 'An error code to identify the error programmatically.',
       type: 'string',
       readOnly: true,
+      example: 'VALIDATION_ERROR',
     },
     message: {
       title: 'Message',
-      description: 'Error message.',
+      description: 'A human readable text to help on error troubleshoot.',
```
Comment (rafael-org) [inline spec/components/schemas/error/base.ts]: Acho que troubleshooting fica melhor.

File: spec/info/description.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 This is the reference documentation for the Zenvia *REST-like* API.
 
+This API is based on resources witch are represented by JSON format and are manipulated using HTTP protocol.
```
Comment (rafael-org) [inline spec/info/description.md]: Tem bruxa aqui não kkkk é which em vez witch.

File: spec/info/description.md [hunk 3]
```diff
@@ -27,51 +27,23 @@ Before use this API you need the following prerequisites:
 * **API Token**: create one API token on [API console](https://app.zenvia.com/home/api)
 * **Webhook** (optional): to receive messages you need to subscribe for this events using the [subscriptions API resources](#tag/Subscriptions)
 
+You can use [sandbox](#section/Getting-started-with-Sandbox) to start and test this API immediately.
```
Comment (rafael-org) [inline spec/info/description.md]: "to start" o que? Talvez "start using"?

File: spec/info/description.md (hunk 2 above)
Comment (hmagarotto) [reply spec/info/description.md]: 🧙‍♀️ rsrs

**Pull request merged**